core/tools.js

```diff
@@ -123,6 +123,42 @@
 	 * @singleton
 	 */
 	CKEDITOR.tools = {
+
+		/**
+		 * Verifies the safety of an html string and returns trusted version.
+		 *
+		 * @param {string} html The safe html string.
+		 * @param {string} justification A short justification for why this
+		 * html string can be verified as safe.
+		 * @returns {TrustedHTML | string} the same html string but as a
+		 * Trusted Type or a string if TT is not supported.
```

Comment:
We can say TrustedHTML here since that's the only Trusted Type we're returning

Reply:
oh sorry, I wasn't clear on the previous comment, I meant let's change the comment to say TrustedHTML instead of "Trusted Type"